internal/manager/featuregates/feature_gates.go

```diff
@@ -26,6 +26,8 @@ const (
 	// objects like Ingress instead of creating a route per path.
 	CombinedRoutesFeature = "CombinedRoutes"
 
+	ExpressionRoutesFeature = "ExpressionRoutes"
```

Comment:
doc comment for this? Ideally with a link to docs explaining what this actually is